.github/workflows/build-lint-test.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        node-version: [12.x, 14.x, 16.x]
+        node-version: [14.x, 16.x]
```

Comment:
If we're updating the minimum supported Node.js version, we should update the version of Node.js referenced in the README, in `package.json`, and in `.nvmrc` as well.

Reply:
aw great catch! I've updated them @Gudahtt thank you!